src/pages/home/sidebar/SidebarScreen/PopoverModal.js

```diff
@@ -0,0 +1,170 @@
+                        },
+                        ...(Permissions.canUsePolicyRooms(this.props.betas) && workspaces.length ? [
+                            {
+                                icon: Expensicons.Hashtag,
+                                text: this.props.translate('sidebarScreen.newRoom'),
+                                onSelected: () => Navigation.navigate(ROUTES.WORKSPACE_NEW_ROOM),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOUSend(this.props.betas) ? [
+                            {
+                                icon: Expensicons.Send,
+                                text: this.props.translate('iou.sendMoney'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_SEND),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOU(this.props.betas) ? [
+                            {
+                                icon: Expensicons.MoneyCircle,
+                                text: this.props.translate('iou.requestMoney'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_REQUEST),
+                            },
+                        ] : []),
+                        ...(Permissions.canUseIOU(this.props.betas) ? [
+                            {
+                                icon: Expensicons.Receipt,
+                                text: this.props.translate('iou.splitBill'),
+                                onSelected: () => Navigation.navigate(ROUTES.IOU_BILL),
+                            },
+                        ] : []),
+                        ...(!Policy.isAdminOfFreePolicy(this.props.allPolicies) ? [
+                            {
+                                icon: Expensicons.NewWorkspace,
+                                iconWidth: 46,
+                                iconHeight: 40,
+                                text: this.props.translate('workspace.new.newWorkspace'),
+                                description: this.props.translate('workspace.new.getTheExpensifyCardAndMore'),
+                                onSelected: () => Policy.createWorkspace(),
+                            },
+                        ] : []),
+                    ]}
+                />
+                <FloatingActionButton
+                    accessibilityLabel={this.props.translate('sidebarScreen.fabNewChat')}
+                    accessibilityRole="button"
+                    isActive={this.state.isCreateMenuActive}
+                    onPress={this.showCreateMenu}
+                />
+            </>
+        );
+    }
+}
+
+PopoverModal.propTypes = propTypes;
+PopoverModal.defaultProps = defaultProps;
+
+export default compose(
+    withLocalize,
+    withWindowDimensions,
+    withOnyx({
+        allPolicies: {
+            key: ONYXKEYS.COLLECTION.POLICY,
+        },
```

Comment:
I don't think it should be done in this PR (this PR is already old enough), but I think it would be great to implement the Onyx property selectors here, like was done in sidebarlinks:

https://github.com/Expensify/App/blob/main/src/pages/home/sidebar/SidebarLinks.js#L221-L224

That could really help with performance here.